packages/amazonq/test/unit/codewhisperer/service/codewhisperer.test.ts

```diff
@@ -14,7 +14,7 @@ import {
 import { globals, getClientId } from 'aws-core-vscode/shared'
 import { AWSError, Request } from 'aws-sdk'
 import { createSpyClient } from 'aws-core-vscode/test'
-import * as os from 'os'
+import { getOperatingSystem } from '../../../../../core/dist/src/shared/telemetry/util'
```

Comment:
This test file lives in core so you'll have to export getOperatingSystem from `src/shard/index.ts` and add getOperatingSystem to the imports of `aws-core-vscode/shared`

Reply:
oh, I see